tests/test_models/test_deadline_ma.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from etna.datasets import TSDataset
+from etna.models.deadline_ma import DeadlineMovingAverageModel
+
+
+@pytest.fixture()
+def df():
+    """Generate dataset with simple values without any noise"""
+    history = 140
+
+    df1 = pd.DataFrame()
+    df1["target"] = np.arange(history)
+    df1["segment"] = "A"
+    df1["timestamp"] = pd.date_range(start="2020-01-01", periods=history)
+
+    df2 = pd.DataFrame()
+    df2["target"] = [0, 2, 4, 6, 8, 10, 12] * 20
+    df2["segment"] = "B"
+    df2["timestamp"] = pd.date_range(start="2020-01-01", periods=history)
+
+    df = pd.concat([df1, df2]).reset_index(drop=True)
+    df = TSDataset.to_dataset(df)
+    tsds = TSDataset(df, freq="1d")
+
+    return tsds
+
+
+@pytest.mark.parametrize("model", [DeadlineMovingAverageModel])
+def test_simple_model_forecaster_run(simple_df, model):
+    sma_model = model(window=1)
```

Comment:
not `sma_model`

Reply:
can you just add `DeadlineMovingAverageModel` to test https://github.com/tinkoff-ai/etna/blob/master/tests/test_models/test_simple_models.py#L13 ?